### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by twofingerrightclick**

### CODE REVIEW

**Comment by markdietzler** [comment]
Not testing for null actor or writer?

**Comment by skozubenko1** [comment]
I like how you break all your work down with comments

File: Inheritance/src/Inheritance/BigBangCastMembers.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+        
+        public void Speak(TextWriter writer)
+        {
+            writer.WriteLine(Name + " says \"Who do we love?\"");
+        }
+    }
+
+
+    public class Raj : Actor
```
Comment (breaman) [inline Inheritance/src/Inheritance/BigBangCastMembers.cs]: Best practices has a separate file per class, these should not have been grouped into one big file.

File: Inheritance/src/Inheritance/BigBangCastMembers.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        public string Name
+        {
+            get { return "Raj"; }
+
+        }
+
+
+        private bool womenPresent= false;
+
+        public bool WomenPresent
```
Comment (breaman) [inline Inheritance/src/Inheritance/BigBangCastMembers.cs]: you could have just made this an auto-implemented property

File: Inheritance/test/Inheritance.Tests/ActorTests.cs [hunk 3]
```diff
@@ -1,13 +1,106 @@
...
+
+            string lineWritten = writerHelper(actor);
+            
+            Assert.AreEqual("Raj mumbles", lineWritten);
+                    
+                
+            
+        }
+
+        public string writerHelper(Actor actor)
```
Comment (breaman) [inline Inheritance/test/Inheritance.Tests/ActorTests.cs]: methods should be defined as PascalCase.

File: Inheritance/test/Inheritance.Tests/PrinterTests.cs [hunk 4]
```diff
@@ -34,9 +34,41 @@ public void PersonGetsPrinted()
                 }
             }
         }
+
+        [TestMethod]
```
Comment (breaman) [inline Inheritance/test/Inheritance.Tests/PrinterTests.cs]: Where is the test for Food?

**Pull request closed without merging**